rdm/0004-installation-and-upgrade-workflow.md

```diff
@@ -0,0 +1,164 @@
+- Start Date: 2019-10-01
+- RFC PR: [#4](https://github.com/inveniosoftware/rfcs/pull/4)
+- Authors: Pablo Panero
+
+# Installation and upgrade workflow
+
+## Summary
+
+Overall design of the installation and upgrade workflows of Invenio RDM.
+
+## Motivation
```

Comment:
is this thought only for developers? what about sysadmins?
If I am an institution sysadmin and I simply want to install InvenioRDM as it is. How do I do so?
By install, I mean in a server. Not for development purposes. Is this in the scope of this RFC? Just to setup my brain for the rest of it :)

Reply:
The RFC should cover both: a developer that wants to run it locally and a sysadmin (what I often call a 'hoster') that just wants to generate a working repo from the get-go. The main difference will be between main application running in containers behind uwsgi (production) or running on host filesystem (development).

Reply:
um...ok, then the motivation list should include some `As a sysadmin...` :)